src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/RemoveAzureRmDataLakeStoreVirtualNetworkRule.cs

```diff
@@ -0,0 +1,69 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.DataLakeStore.Models;
+using Microsoft.Azure.Commands.DataLakeStore.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.DataLake.Store.Models;
+using Microsoft.PowerShell.Commands;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.DataLakeStore
+{
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DataLakeStoreVirtualNetworkRule", SupportsShouldProcess = true), OutputType(typeof(bool))]
```

Comment:
@lewu-msft same comment about adding the parameter set with the `-ResourceId` parameter.

In addition, you'll want to add a parameter set with an `-InputObject` parameter to allow users to pipe from the other cmdlets returning a virtual network rule object to this cmdlet.

For more information about this scenario, please see [the piping scenario here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/piping-in-powershell.md#using-the-inputobject-parameter) and [the code sample of a `Remove-*` cmdlet here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/patterns-for-standard-cmdlets.md#c-example-2).